lib/ansible/plugins/inventory/netbox.py

```diff
@@ -39,6 +39,12 @@
                   region, site, role, platform, and/or tenant. Please check official netbox docs for more info.
             default: False
             type: boolean
+        interfaces:
+            description:
+                - If True, it adds the device or virtual machine interface information in host vars.
+            default: False
+            type: boolean
+            version_added: "2.8"
```

Comment:
please adjust the version to `version_added: "2.9"`

Reply:
version_added bumped. Thanks for your review